inc/testrunnerswitcher.h

```diff
@@ -223,7 +223,7 @@ namespace Microsoft \
             {  \
                 if((q<0)||(size_t)q>=sizeof(EnumName##_Strings)/sizeof(EnumName##_Strings[0])) \
                 { \
-                    return(L"out of range value for " L#EnumName); \
+                    return(L"out of range value for " L"" #EnumName); \
```

Comment:
>L"" [](start = 54, length = 3)

Couldn't it be just without this extra L"" then?

Reply:
I would assume it could. let me check.

Reply:
Is equally good.